app/javascript/entrypoints/helper.js

```diff
@@ -0,0 +1,4 @@
+// eslint-disable-next-line import/prefer-default-export
+export function setTargetHtml(event, html) {
```

Comment:
👍 I am in no way opposed to this or critical, but was this separated in order to decouple from jQuery? I can follow this pattern if it is preferred, but I just wish to be consistent should I implement JS elsewhere.

Reply:
Fuller discussion in standup. For reference, it's not about decoupling jQuery, it's about moving testable functions/classes out of application.js and into modules that just export with no side effects.